src/routes/admin/events/+page.svelte

```diff
@@ -0,0 +1,175 @@
+			name: 'FIRST EVENT',
+			totalSupply: '2,306',
+			transferrable: false,
+			url: 'https://www.google.com.uy/',
+			verifiers: [],
+
+			// Added by Chino
+			eventType: EventTypeEnum.Conference
+		},
+		{
+			claimable: true,
+			dateCreated: '06/05/2023',
+			description: 'This is the second mock',
+			eventId: '34',
+			extraMetadata: { key: 'string' },
+			groups: ['TRY'],
+			host: 'Twitter',
+			image:
+				'https://cdn.discordapp.com/attachments/1054775421671055390/1105958725711319201/tsnakejake_A_cartoon_man_reading_a_mystical_book_with_an_emeral_d5f03067-6692-4152-8ade-37621c0776b5.png',
+			name: 'SECOND EVENT',
+			totalSupply: '2,306',
+			transferrable: false,
+			url: 'https://www.google.com.uy/',
+			verifiers: [{ dateStart: '06/05/2023', dateEnding: '06/20/2023' }],
+
+			// Added by Chino
+			eventType: EventTypeEnum.Workshop
+		},
+		{
+			claimable: true,
+			dateCreated: '06/05/2023',
+			description: 'This is the third mock',
+			eventId: '12',
+			extraMetadata: { key: 'string' },
+			groups: ['TRY'],
+			host: 'Twitter',
+			image:
+				'https://cdn.discordapp.com/attachments/1054775421671055390/1105958725711319201/tsnakejake_A_cartoon_man_reading_a_mystical_book_with_an_emeral_d5f03067-6692-4152-8ade-37621c0776b5.png',
+			name: 'THIRD EVENT',
+			totalSupply: '2,306',
+			transferrable: false,
+			url: 'https://www.google.com.uy/',
+			verifiers: [{ dateStart: '06/05/2023', dateEnding: '06/20/2023' }],
+
+			// Added by Chino
+			eventType: EventTypeEnum.Online
+		}
+	];
+
+	$: searchEvent = events.map((example) => ({
+		...example,
+
+		searchTerms: `${example.name} ${example.eventId}`
+	}));
+
+	$: searchStore = createSearchStore(searchEvent);
+
+	$: unsubscribe = searchStore.subscribe((model) => searchHandler(model));
+
+	onDestroy(() => {
+		unsubscribe();
+	});
+</script>
+
+<div class="main-wrapper" in:fly={{ x: 10, duration: 400 }}>
+	<div class="row-4">
+		<Button color="neutral">Show inactive</Button>
+		<input type="text" placeholder="Search event name or id" bind:value={$searchStore.search} />
+		<div class="row-2">
+			<div class={`button-wrapper ${selectedButton === 'layout' ? 'selected' : 'unselected'}`}>
+				<Button type="transparent" on:click={() => handleButtonClick('layout')}>
+					<Icon icon="basil:layout-outline" color="var(--clr-heading-inverse)" />
+				</Button>
+			</div>
+			<div class={`button-wrapper ${selectedButton === 'list' ? 'selected' : 'unselected'}`}>
+				<Button type="transparent" on:click={() => handleButtonClick('list')}>
+					<Icon icon="ic:round-list" color="var(--clr-heading-inverse)" />
+				</Button>
+			</div>
+		</div>
+		<Button href="/event-generator"><Icon icon="ep:circle-plus" />Create New</Button>
+	</div>
+	{#if selectedButton === 'layout'}
+		<div class="events-wrapper" in:fly={{ x: 10, duration: 400 }}>
+			{#each $searchStore.filtered as event}
+				<EventTicket {event} />
```

Comment:
Also the naming of this components could be a bit better!

For example:
- EventCardList
- EventCardGrid

They are not Tickets!! Tickets are the FLOATs you claim